src/lib/components/textarea/TextArea.component.js

```diff
@@ -0,0 +1,95 @@
+//@flow
+import React from "react";
+import styled, { css } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import { mergeTheme } from "../../utils";
+
+type Props = {
+  rows?: int,
+  cols?: int,
+  placeholder?: string,
+  label?: string,
+  value?: string,  
+  error?: string,
+  disabled?: boolean,
+  onChange: () => void
+};
+
+const TextAreaContainer = styled.div`
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  flex-direction: column;
+
+  ${props => {      
+    console.log("defaultTheme: ", defaultTheme)
+    return props.disabled
+      ? css`
+          cursor: default;
+          opacity: 0.5;
+        `
+      : css`
+          cursor: pointer;
+        `;
+    }
+  }
+  textarea{
+    margin: ${defaultTheme.padding.base};
+    padding: ${defaultTheme.padding.base};
+    border-radius: 10px;
+  }
+  ${props => {
+      const brandingTheme = mergeTheme(props.theme, defaultTheme);
+      return (
+        css`
+        background: ${brandingTheme.backgroundContrast1};
+        textarea{
+          &:hover {
+            border-color: ${brandingTheme.primary};
+          }
+        }
+    ` )
+    }
+  }
+`;
+
+const LabelStyle = styled.label`
+  font-size: ${defaultTheme.fontSize.large};
+  padding-left: ${defaultTheme.padding.base};
+  vertical-align: middle;
+  color: ${props => mergeTheme(props.theme, defaultTheme).text};
+`;
+
+const TextareaErrorMessage = styled.span`
+  color: ${props => mergeTheme(props.theme, defaultTheme).danger};
```

Comment:
since you are using this: `mergeTheme(props.theme, defaultTheme)` a couple times why dont you assign its value to a a variable instead of multiple function calls?
```js
const brandingTheme = mergeTheme(props.theme, defaultTheme); // and use this in all other functions

const { text, danger, primary } = mergeTheme(props.theme, defaultTheme) //or already destructuring
```

and then just call the variable where you need it :ok_woman: